frontend/package.json

```diff
@@ -1,85 +1,86 @@
 {
```

Comment:
Not really sure what you changed here but try to avoid these kinds of changes that seem unnecessary, especially with the package.json file